src/palace/manager/reporting/reports/library_collection.py

```diff
@@ -368,8 +369,7 @@ def _run_report(self, *, session: Session) -> bool:
         # Send success notification.
         self.send_success_notification(access_url=location)
         self.log.info(
-            f"Emailed notification for report '{self.title}' ({self.key}) for "
-            f"library {library.name} ({library.short_name}) to {self.email_address}."
+            f"Creating report '{self.key}' for {library.name} ({library.short_name})."
```

Comment:
Minor: This log message seems misleading (and is a duplicate of line 347). I'd suggest changing "Creating" to "Completed".